rust/Cargo.toml

```diff
@@ -23,7 +23,16 @@ hex = "0.4.3"
 itertools = "0.11.0"
 lazy_static = "1.4.0"
 thiserror = "1.0.41"
+
+# CLI
 anyhow = { version = "1.0.71", optional = true }
```

Comment:
I haven't figured out if it does any harm to users of bumble as a library to have dependencies that are only used by a binary, so until that's sorted out, I'm using a default-enabled feature for these so that library users can turn them off if needed.

Reply:
After some experimentation, it looks like the cost is compile time, but if the user enables LTO, there's no binary size impact. Nonetheless, since I expect most users will be using the CLI via a downloaded binary, it's not much of a hassle to make the feature non-default so that lib users aren't affected at all.